x-pack/plugins/serverless_search/public/application/components/overview.test.tsx

```diff
@@ -51,7 +51,7 @@ describe('<Overview />', () => {
   });
   test('api key', () => {
     const { getByRole } = render(<Overview />);
-    expect(getByRole('heading', { level: 2, name: 'Prepare an API Key' })).toBeDefined();
+    expect(getByRole('heading', { level: 2, name: 'API Key' })).toBeDefined();
```

Comment:
I _think_ this should be 
```suggestion
    expect(getByRole('heading', { level: 3, name: 'Add an API Key' })).toBeDefined();
```

Or maybe just remove `level: 2` completely since the other tests dont use that.